cmd/svcat/binding/bind_cmd.go

```diff
@@ -65,10 +76,11 @@ func NewBindCmd(cxt *command.Context) *cobra.Command {
 		"The name of the secret. Defaults to the name of the instance.",
 	)
 	cmd.Flags().StringSliceVarP(&bindCmd.rawParams, "param", "p", nil,
-		"Additional parameter to use when binding the instance, format: NAME=VALUE")
+		"Additional parameter to use when binding the instance, format: NAME=VALUE. Cannot be combined with --params-json")
 	cmd.Flags().StringSliceVarP(&bindCmd.rawSecrets, "secret", "s", nil,
 		"Additional parameter, whose value is stored in a secret, to use when binding the instance, format: SECRET[KEY]")
-
+	cmd.Flags().StringVar(&bindCmd.jsonParams, "params-json", "",
+		"Additional parameters to use when binding the instance, provided as a JSON object. Cannot be combined with --param")
```

Comment:
We should have a message (fine to be in a follow up) warning people not to use this for secret information.